packages/calypso-build/webpack/sass.js

```diff
@@ -36,10 +37,17 @@ module.exports.loader = ( { includePaths, prelude, postCssOptions } ) => ( {
 		{
 			loader: require.resolve( 'sass-loader' ),
 			options: {
+				implementation: require( 'sass-embedded' ),
+				api: 'modern',
```

Comment:
Maybe we could get better performance by upgrading to a newer version of `sass-loader` and using [the `'modern-compiler'` option](https://github.com/webpack-contrib/sass-loader?tab=readme-ov-file#api) here instead, which enables Sass's [Shared Resources](https://github.com/sass/sass/blob/main/accepted/shared-resources.d.ts.md)?

Reply:
I tried this today and unfortunately I think it only made things worse 😅 I'm wondering if there might be some incompatibility between how the Sass loader is invoked while the JavaScript is being parsed (maybe even relevant to @tyxla's comment about how we're threading elsewhere).

Reply:
Same thing happened to me!